ludwig/schema/metadata/configs/trainer.yaml

```diff
@@ -382,6 +382,22 @@ ecd:
             through the dataset to converge while others converge before a single epoch
             through the data.
         ui_display_name: Train Steps
+    eval_steps:
+        default_value_reasoning:
+            The default value is None because we do not want to lower the number of evaluation steps
+            by default, and we do not know the size of an arbitrary dataset.
+            By setting the default to None, we simply evaluate on the full evaluation set.
+        description_implications:
+            The smaller this value of this parameter, the less time evaluation will take.
```

Comment:
nit: would also mention that smaller values make the evaluation metrics less precise.